onnx/backend/test/case/node/meansquareerror.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import absolute_import
```

Comment:
since some of the logic repeats in the tests you could add a function mse(input, target, reduction='mean', w=None) to call in each of the test. Something like :

def mse(input, target, reduction='mean', w=None):
  out = np.square(input - target)
  if w is not None:
    out = out * w
  if reduction == 'mean':
    out = out.mean()
  elif reduction == 'sum':
    out = out.sum()
 return out